### ISSUE TITLE
Region Screen is broken in a VMWARE workstation VM

### ISSUE THREAD

**Issue by nickallevato** [digitalcreations/MaxTo#510]
**Describe the bug**
When setting regions, the screen is broken as in the screenshot.

**To Reproduce**
Use VMWare.
Latest/update windows.
Latest MaxTo Alpha
Set Regions

**Expected behavior**
Normal region screen 

**Screenshots**

**System information:**
 - Windows version: 
 - MaxTo version: Alpha

**Additional context**
![35000011575316835](https://user-images.githubusercontent.com/6791120/69993727-a415c300-1509-11ea-8349-04f943f2469d.png)

**Comment by vegardlarsen** [comment]
Thank you for the report. Are you using VMWare as the host or client here (and where is MaxTo running)? What display settings are you running?

**Comment by nickallevato** [comment]
The machine running MaxTo is the 'guest;' it is itself a VM, running in VMWare Workstation 15.5.
3840 x 2160
 
![10110011575400270](https://user-images.githubusercontent.com/6791120/70081574-11872980-15c6-11ea-974e-2bdf1248d41f.png)

**Comment by nickallevato** [comment]
I still have this issue with the latest version of MaxTo.
I will try full removals and reinstalls.

**Comment by vegardlarsen** [comment]
Do you only have one monitor? Are you running any other software to make customizations to Windows?

**Comment by nickallevato** [comment]
thanks for the time paying attention to me and my issue.
To be clear, this is a Virtual Machine running inside VMWare Workstation 15.5.1
This virtual machine has a single "monitor". It happens if it is fulls screen or any other resolution.

**Comment by vegardlarsen** [comment]
Is there anything else that may be "strange"? What DPIs are the host computer and the guest computer running at?

**Comment by nickallevato** [comment]
nothing strange about it to begin with
I'll do a fresh VM to prove/disprove the theory

for what it's worth, it was working fine before the last 'large' windows upgrade that broke the previous version(s).
so I'd guess it's something there.

**Comment by nickallevato** [comment]
different VM
![image](https://user-images.githubusercontent.com/6791120/72285223-f1ed1300-35ff-11ea-81fe-2744b041c7b5.png)

**Comment by vegardlarsen** [comment]
Thank you for testing in a different VM. Reproducability is great. I will be looking into this.

**Comment by nickallevato** [comment]
appreciate it
I can try another combination of things if it will help you in any way
might just do a fresh VM you can remote into or something

**Comment by nickallevato** [comment]
i renamed this to be more appropriate

**Comment by vegardlarsen** [comment]
@nickallevato I have tried to reproduce this today, but unfortunately VMWare Workstation will not run on my computer. And being currently locked to working from home because of the corona virus; this will unfortunately have to wait for a while more.

**Comment by nickallevato** [comment]
ironic
also microsoft has released fancyzones powertool which feels like a 5 dollar ripoff of your product, but it does work surprisingly well
stay safe in there, we locked up travel which helped a lot

**Comment by vegardlarsen** [comment]
OK, so 2 BIOS updates, a few BIOS settings, and diagnosing why Windows' wouldn't boot later, I have finally been able to run VMWare Workstation Pro on my machine. I cannot reproduce this exactly, but I did find some irregularities. I am testing with MaxTo 2.1.0-alpha.4 on Windows 10 1909 as the guest machine.

MaxTo does not seem to pick up when you resize the VMWare window. My VM initially started out at 1024x768, and when I resized the entire machine (causing the resolution in the guest to change), MaxTo did not pick up this change, and believe the old resolution was still valid until I restarted MaxTo.

This is probably due to a missing event on VMWare's behalf; I will have to investigate this more closely. I can also see how this will look when resizing the guest to be smaller; leaving the MaxTo window to cover a larger area than the desktop. Could there be some software running on your **host** that would resize the VMWare window for you?
